script.js

```diff
@@ -41,9 +43,86 @@ hamburgerBtn.addEventListener('click', () => {
       hamburgerBtn.style.display = 'flex';
     });
   }
+
   closeBtn.addEventListener('click', () => {
     hamburgerBtn.style.display = 'block';
     navMenu.style.display = 'none';
     closeBtn.style.display = 'none';
   });
 });
+
+const projects = [
+  {
+    name: 'Tonic',
+    specialist: ['CANOPY', 'BackEnd Dev', '2015'],
+    projectImage: '<img height="306px" width="100%" src="images/modal-img-tonic.png" alt="project">',
+    description: "Lorem Ipsum has been the industry's  took a galley of type and scrambled it 1960s.",
+    deploymentLink: '<a href="https://zbahati.github.io/Portfolio/">see live</a>',
+    sourceCode: '<a href="https://github.com/zbahati/Portfolio">see source</a>',
+    tags: ['HTML', 'CSS', 'JavaScript'],
+  },
+  {
+    name: 'Tonics',
+    specialist: ['CANOPY', 'BackEnd Dev', '2015'],
+    projectImage: '<img height="306px" width="100%" src="images/workedit-3.png" alt="project">',
+    description: "Lorem Ipsum has been the industry's  took a galley of type and scrambled it 1960s.",
+    deploymentLink: '<a href="https://zbahati.github.io/Portfolio/">See live</a>',
+    sourceCode: '<a href="https://github.com/zbahati/Portfolio">see source</a>',
+    tags: ['HTML', 'CSS', 'JavaScript'],
+  },
+  {
+    name: 'Multi-Post Stories',
+    specialist: ['CANOPY', 'BackEnd Dev', '2015'],
+    projectImage: '<img src="images/workedit-2.png" height="306px" width="100%" alt="project">',
+    deploymentLink: '<a href="https://zbahati.github.io/Portfolio/">See live</a>',
+    sourceCode: '<a href="https://github.com/zbahati/Portfolio">See source</a>',
+    description: "Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, and typesetting industry.",
+    tags: ['HTML', 'CSS', 'JavaScript'],
+  },
+  {
+    name: 'Multi-Posts Stories',
+    specialist: ['CANOPY', 'BackEnd Dev', '2015'],
+    projectImage: '<img src="images/workedit-4.png" height="306px" width="100%" alt="project">',
+    deploymentLink: '<a href="https://zbahati.github.io/Portfolio/">see live</a>',
+    sourceCode: '<a href="https://github.com/zbahati/Portfolio">see source</a>',
+    description: "Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, and typesetting industry.",
+    tags: ['HTML', 'CSS', 'JavaScript'],
+  },
+
+];
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, there's one important thing that's missing from the requirements:

![image](https://user-images.githubusercontent.com/77069011/233493495-91295928-9e64-49f4-9055-7dd07ad5a918.png)

You still need to update the main page, so the projects section is created dynamically using this information, this will help you have consistent data between the project cards and the detailed popup window! 🤓 🔧 

💡 Hint: Delete the projects section contents in your HTML, just leave the container, then, create a function that iterates over your `projects` array, and for each `project` append the HTML for a `project card` inside the `projects section container`. You can call this function using an event listener on the [`DOMcontentLoaded`](https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event) event to render this section dynamically once the DOM loads.